lean4-infoview/src/infoview/index.css

```diff
@@ -78,25 +91,24 @@ select {
     border-color: var(--vscode-dropdown-border);
 }
 
-/* Styles for the tooltips shown in the widget view. */
+/* Style for tooltips shown in the widget view. This should *not* style the tooltip contents.
+For that, we can use more specific classes. */
 .tooltip {
-    /* color: var(--vscode-editorWidget-foreground);*/
-    @extend .mtk1;
```

Comment:
Note: same goes for `mtk` classes, they are not present in the webview.

Reply:
yeah I wasn't sure about that one, thanks.